apps/api/v2/src/modules/organizations/services/event-types/organizations-event-types.service.ts

```diff
@@ -145,4 +147,17 @@ export class OrganizationsEventTypesService {
 
     return this.eventTypesRepository.deleteEventType(eventTypeId);
   }
+
+  async deleteUserTeamEventTypesAndHosts(userId: number, teamId: number) {
+    try {
+      await this.organizationEventTypesRepository.deleteUserManagedTeamEventTypes(userId, teamId);
+      await this.organizationEventTypesRepository.removeUserFromTeamEventTypesHosts(userId, teamId);
+    } catch (err) {
+      this.logger.error("Could not remove user from all team event-types.", {
+        error: err,
+        userId,
+        teamId,
+      });
+    }
+  }
```

Comment:
silently removing user from all team event-types, we don't want to trigger an error here since it's a side effect of removing a membership